apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/payment/payu/api/PayUController.java

```diff
@@ -0,0 +1,35 @@
+package edu.pw.apsienrollment.payment.payu.api;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.pw.apsienrollment.payment.PaymentService;
+import edu.pw.apsienrollment.payment.payu.api.dto.NotificationDto;
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequestMapping("payment/payu")
+@RequiredArgsConstructor
+@Validated
+public class PayUController {
+    private final PaymentService paymentService;
+
+    @PostMapping("notify")
+    ResponseEntity<Void> notify(@RequestBody NotificationDto notificationDto) {
+        System.out.printf("[%s]\n", notificationDto.getOrder().getStatus());
+        System.out.printf("[%s]\n", notificationDto.getOrder().getExtOrderId());
+
+        if (notificationDto.getOrder().getStatus().equals("COMPLETED")) {
+            System.out.println("!!!");
```

Comment:
Please, remove this println or replace it with a more informative log

Reply:
removed